Models/articles.models.js

```diff
@@ -1,10 +1,16 @@
 const db = require('../db/connection')
 
 exports.fetchArticles = (req) => {
+
     const articlesId = req.params.article_ID
+
     return db.query(`SELECT * FROM articles WHERE article_id=${articlesId}`).then(({rows}) =>{
```

Comment:
You are open to SQL injection here, use $1 instead and pass the id into an array